kafka/producer.c

```diff
@@ -124,7 +124,7 @@ add_producer_topics(producer_topics_t *topics, rd_kafka_topic_t *element) {
     return 0;
 }
 
-rd_kafka_topic_t *
+static rd_kafka_topic_t *
```

Comment:
Why are you using the `static` word here?

Reply:
This function isn't used outside current file.